docs/topic/troubleshooting.md

```diff
@@ -0,0 +1,28 @@
+# Troubleshooting and debugging
+
+These sections describe a few common and useful tips for troubleshooting our infrastructure.
+
+## Roll back (revert) a helm deploy
+
+Sometimes it is useful to simply **revert** a Kubernetes deployment with Helm.
+For example, if you've manually deployed something via `helm install`, and notice that something is wrong with our deployment.
```

Comment:
```suggestion
For example, if you've manually deployed something via `helm upgrade`, and notice that something is wrong with our deployment.
```

`helm upgrade` is the much more likely command we will be using a rollback to recover from. `helm install` implies that the chart wasn't deployed before the command was run, therefore there's nothing to rollback to. A "fix" in that scenario is to just delete the deployment.